package-lock.json

```diff
@@ -1,7239 +0,0 @@
-{
-	"name": "repo-automation-bots",
-	"requires": true,
-	"lockfileVersion": 1,
-	"dependencies": {
```

Comment:
I'm open to getting rid of `package-lock.json` files, but I think in practice, given that we're deploying apps they would be a good idea (if we weren't seeing drift in TypeScript definitions so frequently).

Reply:
In general, I would agree. But it's causing __pain__.